modules/home/assets/js/components/home-screen.js

```diff
@@ -7,14 +7,16 @@ const HomeScreen = ( props ) => {
 		embedUrl = 'https://www.youtube.com/embed/icTcREd1tAg?si=40E8D1hdnu26-TXM';
 
 	return (
-		<Container sx={ { display: 'flex', flexDirection: 'column', gap: { xs: 1, md: 3 }, py: { xs: 2, md: 6 }, maxWidth: { md: '990px' } } }>
-			<TopSection
-				videoUrl={ videoUrl }
-				embedUrl={ embedUrl }
-				topData={ props.homeScreenData.data.top[ 0 ] }
-				createNewPageUrl={ props.homeScreenData.createNewPageUrl }
-			/>
-		</Container>
+		<Box maxWidth="xl" sx={ { pr: 1 } }>
```

Comment:
Maybe can you add a comment here about the reason why we have added this?

Reply:
Can you please move this comment to line 10, so that it is immediately linked to the` <Box>` component.